pyhomematic/devicetypes/thermostats.py

```diff
@@ -151,6 +177,12 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         self.SENSORNODE.update({"TEMPERATURE": [1],
                                 "HUMIDITY": [1]})
         self.WRITENODE.update({"SETPOINT": [2]})
+        # Remove invalid parameters
+        self.ATTRIBUTENODE.pop("BATTERY_STATE", None)
+        self.ATTRIBUTENODE.pop("CONTROL_MODE", None)
+        self.SENSORNODE.pop("ACTUAL_TEMPERATURE", None)
+        self.SENSORNODE.pop("ACTUAL_HUMIDITY", None)
+        self.WRITENODE.pop("SET_TEMPERATURE", None)
```

Comment:
Huh, true. I just recall seeing those parameters in the HASS logs. But that was probably because it was hardcoded there.